### ISSUE TITLE
How to create rarity for a layer group?

### ISSUE THREAD

**Issue by 48bits** [HashLips/hashlips_art_engine#1357]
How do I config the chance of a layer group being generated? 

(eg. I only want 50% of the image to have a background)

**Comment by bolshoytoster** [comment]
@48bits you could have two `layerConfigurations`; one with and one without the background.

**Comment by 0xinit** [comment]
@bolshoytoster what are more usecases of `layerConfigurations` variable? The video doesn't explain very well... but if a trait for example "red eyes" is considered in 2nd layerConfigurations array, then how would it affect the rarity of "red eyes" in the first layerConfiurations?

**Comment by bolshoytoster** [comment]
@0xinit I don't think it will affect therarity in the first one, but could you give an example `layerConfigurations`?
